platforms/r3-corda/configuration/roles/create/storageclass/tasks/main.yaml

```diff
@@ -24,6 +28,7 @@
 - name: "Push the created deployment files to repository"
   include_role: 
     name: "{{ playbook_dir }}/../../shared/configuration/roles/git_push"
+  tags: notest
```

Comment:
use tags as a list so that new tags can be added
- notest

Reply:
Sure